```
/Users/buildnode/jenkins/workspace-private/swift-PR-source-compat-suite/swift/lib/ClangImporter/ClangImporter.cpp:2464:41: error: use of undeclared identifier 'SerializedSwiftName'
  for (auto entry : lookupTable->lookup(SerializedSwiftName(name.str()),
                                        ^
```

Right, this needs a bit of backporting. My bad.

--- a/lib/ClangImporter/ClangImporter.cpp
+++ b/lib/ClangImporter/ClangImporter.cpp
@@ -2461,7 +2461,7 @@ ClangModuleUnit::lookupNestedType(Identifier name,
   // FIXME: This is very similar to what's in Implementation::lookupValue and
   // Implementation::loadAllMembers.
   SmallVector<TypeDecl *, 2> results;
-  for (auto entry : lookupTable->lookup(SerializedSwiftName(name.str()),
+  for (auto entry : lookupTable->lookup(name.str(),
                                         baseTypeContext)) {
     // If the entry is not visible, skip it.
     if (!isVisibleClangEntry(clangCtx, entry)) continue;

